Makefile

```diff
@@ -20,6 +20,9 @@ system:
 	for extension in $(SystemJS); do \
 		echo $$extension"(System);" >> dist/$@.src.js; \
 	done
+	for extension in $(SystemJS); do \
+		echo "System._extensions.push("$$extension");" >> dist/$@.src.js; \
+	done
```

Comment:
Now that core is applied first, we may be able to define `System._extensions` in core?